Source/dvlnet/frame_queue.cpp

```diff
@@ -56,7 +57,7 @@ bool frame_queue::PacketReady()
 		if (Size() < sizeof(framesize_t))
 			return false;
 		auto szbuf = Read(sizeof(framesize_t));
-		std::memcpy(&nextsize, &szbuf[0], sizeof(framesize_t));
+		nextsize = LoadLE32(szbuf.data());

```

Comment:
Should we add an assert that checks that framesize_t is 32bit?`

Reply:
Done